stacks/java-quarkus/1.5.0/devfile.yaml

```
@@ -0,0 +1,70 @@
+schemaVersion: 2.2.0
+metadata:
+  name: java-quarkus
+  displayName: Quarkus Java
+  description: Java application using Quarkus and OpenJDK 17
```

I would correct the mentioned OpenJDK version to the one bumped in these changes:

```suggestion
  description: Java application using Quarkus and OpenJDK 21
```